src/plone/api/content.py

```diff
@@ -652,4 +652,7 @@ def find(context=None, depth=None, **kwargs):
     if not valid_indexes:
         return []
 
-    return catalog(**query)
+    if kwargs.get("unrestricted"):
```

Comment:
I would update the function signature to have `unrestricted=False` as a default, otherwise you will be passing `unrestricted` to the catalog search.